lib/node_modules/@stdlib/complex/float64/base/add/README.md

```diff
@@ -123,22 +123,17 @@ The function supports the following parameters:
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var Complex128 = require( '@stdlib/complex/float64/ctor' );
-var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var Complex128Array = require( '@stdlib/array/complex128' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var logEachMap = require( '@stdlib/console/log-each-map' );
 var add = require( '@stdlib/complex/float64/base/add' );
 
-var rand = discreteUniform( -50, 50 );
-
-var z1;
-var z2;
-var z3;
-var i;
-for ( i = 0; i < 100; i++ ) {
-    z1 = new Complex128( rand(), rand() );
-    z2 = new Complex128( rand(), rand() );
-    z3 = add( z1, z2 );
-    console.log( '(%s) + (%s) = %s', z1.toString(), z2.toString(), z3.toString() );
-}
+// Generate arrays of random values:
+var z1 = new Complex128Array( discreteUniform( 200, -50, 50 ) );
+var z2 = new Complex128Array( discreteUniform( 200, -50, 50 ) );
+
+// Perform multiplication on each element of arrays:
```

Comment:
@Planeshifter Please perform more careful reviews. This comment is clearly incorrect.